packages/application-generic/src/services/queues/queue-base.service.ts

```diff
@@ -71,52 +71,60 @@ export class QueueBaseService {
     await this.gracefulShutdown();
   }
 
-  public async addMinimalJob(
-    id: string,
-    data?: any,
-    groupId?: string,
-    options: JobsOptions = {}
-  ) {
+  public async addMinimalJob(params: IJobParams) {
```

Comment:
Refactored to get object instead of params list so we could be explicit about the params.